.github/workflows/net-perf-gke.yaml

```diff
@@ -0,0 +1,252 @@
+name: Network performance GKE
+
+# Any change in triggers needs to be reflected in the concurrency group.
+on:
+  schedule:
+    - cron: '39 0 * * 1-5'
+
+# By specifying the access of one of the scopes, all of those that are not
+# specified are set to 'none'.
+permissions:
+  # To be able to access the repository with actions/checkout
+  contents: read
+
+concurrency:
+  # Structure:
+  # - Workflow name
+  # - Event type
+  # - A unique identifier depending on event type:
+  #   - schedule: SHA
+  #   - workflow_dispatch: PR number
+  #
+  # This structure ensures a unique concurrency group name is generated for each
+  # type of testing, such that re-runs will cancel the previous run.
+  group: |
+    ${{ github.workflow }}
+    ${{ github.event_name }}
+    ${{
+      (github.event_name == 'schedule' && github.sha) ||
+      (github.event_name == 'workflow_dispatch' && github.event.inputs.PR-number)
+    }}
+  cancel-in-progress: true
+
+env:
+  clusterName: ${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}-${{ github.run_attempt }}
+  cilium_cli_ci_version:
+  CILIUM_CLI_MODE: helm
+  test_name: gke-perf
+  check_url: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
+  USE_GKE_GCLOUD_AUTH_PLUGIN: True
+  gcp_zone: us-east5-a
+  k8s_version: 1.28
+
+jobs:
+  installation-and-perf:
+    name: Installation and Perf Test
+    runs-on: ubuntu-latest
+    timeout-minutes: 40
```

Comment:
Maybe we should bump this to 60? The wait for images step has a timeout of 30 minutes, meaning this line implies that all the other steps should take no more than 10 minutes. In a worst-case scenario where waiting for images takes 20 minutes, having some extra buffer room would be nice.

Reply:
Sounds good. Bumped to 60.
FYI currently workflow takes ~30 minutes in total of which performing the test is 20 minutes.